### PR TITLE
GitHub workflow: Migrate CreatePullRequest to shared action

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Action: https://github.com/SonarSource/gh-action-lt-backlog/tree/master/CreateCardForStandalonePR

Source: https://github.com/SonarSource/gh-action-lt-backlog/blob/master/CreateCardForStandalonePR/CreateCardForStandalonePR.ts

### CODE REVIEW

File: .github/workflows/CreatePullRequest.yml [hunk 1]
```diff
@@ -5,40 +5,17 @@ on:
     types: ["opened"]
 
 jobs:
-  assign_and_create_card:
-    name: Assign issue to sender and create Kanban card
+  CreateCardForStandalonePR_job:
+    name: Assign PR to the author and create a Kanban card
```
Comment (costin-zaharia-sonarsource) [inline .github/workflows/CreatePullRequest.yml:9]: I would replace Kanban with "project" or "GitHub" project

File: .github/workflows/CreatePullRequest.yml [hunk 2]
```diff
@@ -5,40 +5,17 @@ on:
...
-            const pr = context.payload.pull_request;
-            const matches = pr.body == null ? null : pr.body.match(/(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved)\s*#\d+/gi);
-            if(matches) {
-                console.log("Skip, contains '" + matches[0] + "'");
-            } else {
-                addAssignee(pr, context.payload.sender.login);
-                github.projects.createCard({ column_id: IN_PROGRESS_COLUMN, content_id: pr.id, content_type: "PullRequest" });
-                console.log("Done");
-            }
+          column-id: 4971952
```
Comment (costin-zaharia-sonarsource) [inline .github/workflows/CreatePullRequest.yml:21]: Would be great to define a constant for this.

**Review by costin-zaharia-sonarsource — CHANGES_REQUESTED** [review]

File: .github/workflows/CreatePullRequest.yml (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/CreatePullRequest.yml:9]: I find "Kanban" more clear about what it really does. While "project" and "GitHub" are technically correct, they sound ambiguous.

File: .github/workflows/CreatePullRequest.yml (hunk 2 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/CreatePullRequest.yml:21]: I prefer to keep it simple as it is. Non-reusable constant doesn't make much sense here. The docs are pretty clear about what value should be provided here.

File: .github/workflows/CreatePullRequest.yml (hunk 1 above)
Comment (costin-zaharia-sonarsource) [reply .github/workflows/CreatePullRequest.yml:9]: > Kanban is a scheduling system for lean manufacturing Taiichi Ohno, an industrial engineer at Toyota, developed kanban to improve manufacturing efficiency.

Kanban is a process, we might follow this or another process in the future but this script is about GitHub projects and I would stick to that. I prefer to be technically correct.

Comment (pavel-mikula-sonarsource) [reply .github/workflows/CreatePullRequest.yml:9]: As long as that board is called Kanban, it's not wrong. And same board is called Kanban also in Jira instances.

File: .github/workflows/CreatePullRequest.yml (hunk 2 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/CreatePullRequest.yml:21]: Added comment

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**